_sass/components/_book-authors.scss

```diff
@@ -0,0 +1,31 @@
+.book-authors {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(20em, 1fr));
+
+  column-gap: var(--padding-md);
+  row-gap: var(--padding-lg);
+}
+
+.book-author {
+  display: grid;
+
+  grid-template-areas:
+    "icon main";
+  grid-template-columns: auto 1fr;
+  column-gap: var(--padding-sm);
+
+  > * {
+    grid-area: main;
+  }
+
+  > p {
+    max-width: 43ch;
+  }
+
+  > .book-author-icon {
+    grid-area: icon;
+    width: 6ch;
+    height: 6ch;
```

Comment:
**suggestion:** `aspect-ratio: 1` is more explicit about the intention. And adding `object-fit: contain;` would future-proof for images that are not square.
```suggestion
    width: 6ch;
    aspect-ratio: 1;
    object-fit: contain;
```

We should probably have an abstraction for a rounded image element. There are several ones already in the codebase. Feel free to introduce that, or I'll take care of it afterwards.